cartography/intel/aws/lambda_function.py

```diff
@@ -0,0 +1,84 @@
+import logging
+
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_lambda_data(boto3_session, region):
+    """
+    Create an Lambda boto3 client and grab all the lambda functions.
+    """
+    client = boto3_sessionclient('lambda', region_name=region)
+    paginator = client.get_paginator('list_functions')
+    lambda_functions = []
+    for page in paginator.paginate():
+        for each_function in page['Functions']:
+            lambda_functions.append(each_function)
+    return ({'AWSLambda': lambda_functions})
+
+
+@timeit
+def load_lambda_functions(neo4j_session, data, region, current_aws_account_id, aws_update_tag):
+    ingest_lambda_functions = """
+    MERGE (lambda:AWSLambda{id: {Arn}})
+    ON CREATE SET lambda.firstseen = timestamp(), lambda.modifieddate = {LastModified}
```

Comment:
Remove `lambda.modifieddate = {LastModified}`; we don't need it there.

Reply:
I was not sure, I was following the other files. I honestly cant tell the difference between ON CREATE SET and SET due to my lack of Neo4j experience even after I read the documentation

Reply:
`on create set` sets that field only if the node is completely brand new.

`set` sets the field every time.